_scripts/instruction-widget/install.js

```diff
@@ -98,16 +99,19 @@ module.exports = function(context) {
     // Debian Jessie
     context.base_command = "certbot";
     context.cron_included = true;
+    context.package = "certbot";
+
     if (context.webserver == "apache") {
       context.package = "python-certbot-apache";
-    } else {
-      context.package = "certbot"
+    } else if (context.webserver == "nginx") {
+      context.certonly = true;
```

Comment:
Yeah, those are in there in all the places that we're not ready to expose it yet due to not being packaged or being too old of a version (see document linked below).

When this is here, it shouldn't be recommending nginx in the installer, unless I've completely misunderstood how this works.